lib/spack/spack/test/spec_dag.py

```diff
@@ -14,6 +16,9 @@
 from spack.spec import Spec
 from spack.util.mock_package import MockPackageMultiRepo
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
```

Comment:
Like tests for concretization, most of these tests confirm behavior that is fundamental, so ideally we can get as many of them running as possible.